packages/nteract-data-explorer/src/index.tsx

```diff
@@ -85,8 +83,13 @@ export class NteractDataExplorerWidget extends Widget implements IRenderMime.IRe
 
         clearInterval(timer);
         ReactDOM.render(
-          // @ts-ignore
-          <DataExplorer data={data} metadata={model.metadata.dataExplorer || {}} onMetadataChange={onMetadataChange} />,
+          <DataExplorer
+            data={data as unknown as Props['data']}
+            metadata={(model.metadata.dataExplorer as unknown as Props['metadata']) || {}}
+            mediaType={MIME_TYPE}
+            initialView={'grid'}
```

Comment:
mediaType and initialView were using default values which are the same as MIME_TYPE and 'grid'?

Reply:
yes, I just copy and paste the default values here because TypeScript asks for these required props.